content/docs/get-started/kubernetes/review-project.md

```diff
@@ -145,18 +145,38 @@ class Program
 This Pulumi program creates an NGINX deployment and exports the name of the deployment.
 
 {{% lang python %}}
+
+> *Note*: As a prerequisite, [install virtualenv](https://virtualenv.pypa.io/en/latest/installation/) to manage project requirements
+
 For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
 
+Create a virtual environment:
+
 ```bash
-$ virtualenv -p python3 venv
+$ python3 -m venv venv
 ```
 
+Activate the environment in Linux and macOS:
+
 ```bash
 $ source venv/bin/activate
 ```
 
+Create a virtual environment in Windows:
+
+```bat
+$ python -m venv venv
+
+Activate the environment in Windows:
+
+```bat
+> venv\Scripts\activate.bat
+```
+
+Install dependencies:
+
 ```bash
-$ pip3 install -r requirements.txt
+$ pip install -r requirements.txt
```

Comment:
Is this supposed to be `pip` or `pip3`? Elsewhere we still have it as `pip3`.

Reply:
It's probably better to leave it as pip3 to remove any ambiguity about if we're actually running in a Python3 environment.